transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -171,6 +208,28 @@ impl SignatureRctBulletproofs {
         )
     }
 
+    /// Return low level signing data for signing in external signers
+    pub fn get_view_only_signing_data<CSPRNG: RngCore + CryptoRng>(
+        block_version: BlockVersion,
+        message: &[u8; 32],
+        input_rings: &[InputRing],
+        output_secrets: &[OutputSecret],
+        fee: Amount,
+        rng: &mut CSPRNG,
+    ) -> Result<ViewOnlySigningData, Error> {
+        let signing_data = get_view_only_signing_data(
```

Comment:
Is it possible to move the contents of this helper method into this one? I am not sure what we get from having this just call the other one.